certified-connectors/Mural/apiDefinition.swagger.json

```diff
@@ -1,1074 +1,1394 @@
+				"parameters": [
+					{
+						"name": "body",
+						"in": "body",
+						"required": false,
+						"schema": {
+							"type": "object",
+							"properties": {
+								"workspaceId": {
+									"type": "string",
+									"description": "Workspace",
+									"title": "Workspace",
+									"x-ms-dynamic-values": {
+										"operationId": "GetWorkspaceOperation",
+										"value-path": "id",
+										"value-collection": "value",
+										"value-title": "name"
+									}
+								},
+								"roomId": {
+									"type": "integer",
+									"description": "Room",
+									"title": "Room",
+									"format": "int64",
+									"x-ms-dynamic-values": {
+										"operationId": "GetRoomsOperation",
+										"value-path": "id",
+										"value-collection": "value",
+										"value-title": "name",
+										"parameters": {
+											"workspaceId": {
+												"parameter": "workspaceId"
+											}
+										}
+									},
+									"x-ms-dynamic-list": {
+										"operationId": "GetRoomsOperation",
+										"itemsPath": "value",
+										"itemValuePath": "id",
+										"itemTitlePath": "name",
+										"parameters": {
+											"workspaceId": {
+												"parameterReference": "body/workspaceId"
+											}
+										}
+									}
+								},
+								"title": {
+									"type": "string",
+									"description": "Title",
+									"title": "Title"
+								}
+							},
+							"required": [
+								"roomId",
+								"workspaceId"
+							]
+						}
+					}
+				]
+			}
+		},
+		"/api/public/v1/murals/{muralId}/widgets/sticky-note": {
+			"post": {
+				"responses": {
+					"201": {
+						"description": "default",
+						"schema": {
+							"type": "object",
+							"properties": {
+								"value": {
+									"type": "object",
+									"properties": {
+										"contentEditedBy": {
+											"type": "object",
+											"properties": {
+												"firstName": {
+													"type": "string",
+													"description": "first name"
+												},
+												"lastName": {
+													"type": "string",
+													"description": "last name"
+												}
+											},
+											"description": "content edited by"
+										},
+										"contentEditedOn": {
+											"type": "integer",
+											"format": "int32",
+											"description": "content edited on"
+										},
+										"createdBy": {
+											"type": "object",
+											"properties": {
+												"firstName": {
+													"type": "string",
+													"description": "first name"
+												},
+												"lastName": {
+													"type": "string",
+													"description": "last name"
+												}
+											},
+											"description": "created by"
+										},
+										"createdOn": {
+											"type": "integer",
+											"format": "int32",
+											"description": "created on"
+										},
+										"height": {
+											"type": "number",
```

Comment:
@rmiller-mural The type has been changed for the output field. Existing flows may be broken if they are using this field and actual payload has been changed. 
can you please confirm that no users will be affected by this change?

Reply:
I reverted the change. We can revisit it in a future release if it becomes an issue.